### DISCUSSION TITLE
Will there be WebDav sync?

### DISCUSSION THREAD

**Discussion by cb3inco** [super-productivity/super-productivity#598] [Q&A] [answered]
sorry, just wondering as it's not totally clear, but is WebDav sync coming to version 6? Is that what I'm seeing? I'd like to sync this with NextCloud (rather than Dropbox/Google Drive) and wasn't sure. Will this be something that will be able to be setup in Settings section?

**Comment by johannesjo** [comment, marked as answer]
Hi there! 

> is WebDav sync coming to version 6?

Yes!

It should be working pretty well already. You can test and probably even use the feature productively with the latest release candidate:
https://github.com/johannesjo/super-productivity/releases/tag/v6.0.0-rc.5

Please let me know what you think about and if you encounter any issues etc.!

**Reply by cb3inco** [reply]
I'm running into this issue:
![image](https://user-images.githubusercontent.com/14948644/96691188-c842f280-13ae-11eb-838c-74c1d8aa5c16.png)

If I try to force it, it just fails. I'm using the desktop app on Linux (Pop_OS)

**Reply by johannesjo** [reply]
Thanks for getting back to me! Could you please open up a [bug ticket](https://github.com/johannesjo/super-productivity/issues/new?assignees=&labels=bug&template=BUG_REPORT.md&title=) with some logs?

**Reply by cb3inco** [reply]
Sure, how do I create logs?

**Reply by johannesjo** [reply]
### Console Output
Is there any output if you press Ctrl+Shift+i (Cmd+Alt+i for mac) in the console tab? If so please post it here.

### Error Log (Desktop only)
For the desktop versions, there is also an error log file in case there is no console output. 
Usually, you can find it here:
on Linux: 
~/.config/superProductivity/log.log
--or--
~/snap/superproductivity/current/.config/superProductivity/log.log

on macOS: ~/Library/Logs/superProductivity/log.log

on Windows: %USERPROFILE%\AppData\Roaming\superProductivity\log.log
